taker-electron/src/main/main.ts

```diff
@@ -132,34 +134,64 @@ app.on("window-all-closed", () => {
     }
 });
```

Comment:
is the `alive` closure still used anywhere, maybe it can be removed? I don't think it does anything sensible (also: `timeout` is not used there, and `port` should be captured as parameter if it was to be used).

Reply:
Yes alive is still used.. It tests periodically if the itchysats service is available.